### ISSUE TITLE
Inconsistent Image Result Sizes

### ISSUE THREAD

**Issue by JakeAi** [CrackerakiUA/ui-cropper#59]
Hello @CrackerakiUA I have been having inconsistent issues with your library. This is literally amazing by the way. But as you can see in the codepen. Everything is set to Square and 1:1 yet the result image is not 1:1
I'm sure the issue is coming down to it being responsive, but.... shouldn't it be able to support responsiveness?

https://codepen.io/anon/pen/MOYOow

**Comment by CrackerakiUA** [comment]
codepen looks to work, how did you manage to make it not 1:1 ?

**Comment by JakeAi** [comment]
@CrackerakiUA I opened the link I commented above. Inspected the Image element. The image is 594.33 x 595.81. When clicking the download button the image produced is 798 x 800. Also on the right in the pre json block:
  "cropImageWidth": 1712,
  "cropImageHeight": 1716,

**Comment by CrackerakiUA** [comment]
ok, i can reproduce that also. Never had problem with such details, need to check why it's not precise.

**Comment by JakeAi** [comment]
@CrackerakiUA the way I am dealing with this now is (below). Even with the area being rectangle and 1:1 its not consistent. I have to force it to square. These are on ng-change events.

```JS
m.changeSize = function() {
    if (m.editImageOptions.resultImageSizeW === m.editImageOptions.resultImageSizeH) {
        m.editImageOptions.aspectRatio = 1;
    }
    m.changeAspect();
};
m.changeAspect = function() {
    if (m.editImageOptions.aspectRatio === 1) {
        m.editImageOptions.areaType = 'square';
    } else {
        m.editImageOptions.areaType = 'rectangle';
    }
};

m.changeCrop = function() {
    if (m.editImageOptions.areaType === 'square' || m.editImageOptions.areaType === 'circle') {
        m.editImageOptions.aspectRatio = 1;
    }
    if (m.editImageOptions.areaType === 'rectangle') {
        m.editImageOptions.aspectRatio = 0;
    }
};
```

**Comment by CrackerakiUA** [comment]
oh, but this should be working on your side, this not helps other scenarios.

**Comment by JakeAi** [comment]
What do you mean? I found an issue and I shouldn't have to apply my fix?

**Comment by CrackerakiUA** [comment]
No, please do, Have you checked all codepens if it fix the issue and work smooth everywhere?

**Comment by JakeAi** [comment]
@CrackerakiUA Sorry, I am confused. I don't have a fix. The only fix I have is the code I posted above. I was waiting to see what you come up with.
